src/validations/signup.validation.js

```diff
@@ -0,0 +1,19 @@
+import Joi from '@hapi/joi';
```

Comment:
Move this file to `auth.validation.js` so it can accommodate other validation middleware to the auth route.